test/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+# Set root logger level to DEBUG and its only appender to A1.
```

Comment:
Lets drop this, we have a method (`setUpLog()`) to configure the logger when really needed.

Reply:
You mean drop the file entirely?

Reply:
Yes, seems unncessary.